app/helpers/podcasts_helper.rb

```diff
@@ -5,6 +5,23 @@ module PodcastsHelper
 
   RSS_LANGUAGE_CODES = %w[af sq eu be bg ca zh-cn zh-tw hr cs da nl nl-be nl-nl en en-au en-bz en-ca en-ie en-jm en-nz en-ph en-za en-tt en-gb en-us en-zw et fo fi fr fr-be fr-ca fr-fr fr-lu fr-mc fr-ch gl gd de de-at de-de de-li de-lu de-ch el haw hu is in ga it it-it it-ch ja ko mk no pl pt pt-br pt-pt ro ro-mo ro-ro ru ru-mo ru-ru sr sk sl es es-ar es-bo es-cl es-co es-cr es-do es-ec es-sv es-gt es-hn es-mx es-ni es-pa es-py es-pe es-pr es-es es-uy es-ve sv sv-fi sv-se tr uk]
 
+  def feed_description(feed, podcast)
+    desc = feed.description.present? ? feed.description : podcast.description
+    desc ||= ""
+    if podcast.has_apple_config?
+      desc = desc.truncate_bytes(4000, omission: "")
+    end
+    desc
+  end
+
+  def episode_description(episode)
+    desc = episode.description_with_default
+    if episode.podcast.has_apple_config?
+      desc = episode.description.truncate_bytes(4000, omission: "")
```

Comment:
Per other comment, `description_with_default` is the non-validated validated output (also in the apple api call).

Reply:
are you thinking we just add the 4k limit to that method?

Reply:
oh, I have an error here, meant to use the desc `episode.description_with_default` value below, will fix

Reply:
Yeah, that's what I was thinking:  To incorporate the transformations into a single method `description_with_default` (that name is drifting out of relevance. `description_for_integrations` ? naming is hard), so that we can couple these "extra" validations together in one place

Reply:
ok, added a `episode#description_safe` method that I think captures the spirit of it!
Also DRY'd up the 4000 value to constant while I was at it